client-cli/src/command.rs

```diff
@@ -12,6 +12,9 @@ use prettytable::{cell, format, row, Cell, Row, Table};
 use quest::success;
 use structopt::StructOpt;
 
+use self::address_command::AddressCommand;
+use self::transaction_command::TransactionCommand;
+use self::wallet_command::WalletCommand;
```

Comment:
Do we have any convention on ordering imported crates? @tomtau @devashishdxt @leejw51crypto

Reply:
cargo format reorders by some rule.
i will check again.

Reply:
yes, cargo format reorders

Reply:
the convention is whatever the default `cargo fmt` rule is :)

Reply:
`cargo fmt` will order alphabetically. But, many Rust projects follow this rule;
```
std imports;
// blank line
external crate imports;
// blank line
self crate imports;
```

Reply:
external is not necessary in latest rust